<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkAdapterForms</class>
 <widget class="QDialog" name="NetworkAdapterForms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetBottomDynamic" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>105</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widgetNavigationDynamic" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnHorseScan">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>105</height>
               </size>
              </property>
              <property name="text">
               <string>网卡信息</string>
              </property>
              <property name="icon">
               <iconset resource="../source.qrc">
                <normaloff>:/image/horse_check.png</normaloff>:/image/horse_check.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="actNetworkAdapterPushButton">
              <property name="text">
               <string>开始扫描</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>797</width>
               <height>554</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>8</number>
              </property>
              <property name="bottomMargin">
               <number>8</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTableWidget" name="tableWidgetNetworkAdapterForms">
                <column>
                 <property name="text">
                  <string>逻辑名</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ipv4</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>MAC</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>设备类型</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>设备型号</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>netmask</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>dns</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>厂商</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>ipv6</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>id</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>物理id</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>总线id</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>version</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>带宽</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>时钟频率</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>性能</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>配置</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>容量</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../source.qrc"/>
 </resources>
 <connections/>
</ui>
